packages/salesforcedx-vscode-core/src/telemetry/telemetryReporter.ts

```diff
@@ -130,6 +130,10 @@ export default class TelemetryReporter extends vscode.Disposable {
       commonProperties['common.vscodemachineid'] = vscode.env.machineId;
       commonProperties['common.vscodesessionid'] = vscode.env.sessionId;
       commonProperties['common.vscodeversion'] = vscode.version;
+      if (vscode.env.uiKind) {
```

Comment:
`vscode.env.uiKind` seems to be an enum based on the docs and the types, do we need this check ? Everything else looks good.

Reply:
Probably not. It's an abundance of caution in case VSCode has somehow left this undefined, in which case using the value as a key in the next line might throw an exception. Want me to get rid of this check?

Reply:
No, I just wanted to understand why you added it. I agree with you, let's leave it as is.